package.json

```diff
@@ -26,6 +33,7 @@
     "gulp-streamify": "^1.0.2",
     "gulp-stylus": "^2.1.0",
     "gulp-uglify": "^1.5.1",
+    "react": "^0.14.7",
```

Comment:
> Now that React-Motion and React-SimpleTabs are dependencies of GeniBlocks, if we don't have the dev-dependency, then on npm install we get

Hmmm... but it still has exactly zero effect on development or building, unless I'm mistaken. We never touch the package at all. It seems to me like if we're adding the package simply to remove the npm warning, that signifies we're doing something wrong.

I think if we bring react in as a package, we should work out how to use it as a package, and not use the bower version. Since we're already building the examples, it seems like this wouldn't be too hard. It doesn't look like we're currently using Browserify on the examples, though, so a simple `import` statement won't work until we change how we're building them.

> The difference now is that the GeniBlocks code can/must require or import React explicitly

This doesn't seem related to the listing of React as a peerDependency or as a devDependency, since we can do this even if we don't list it at all.

> As near as I can tell, this is the model that React-based libraries seem to be settling on

Having read through a bunch of this discussion online, I agree with you on this. And now that I understand that peerDependencies are not brought in (which seems counter to this statement in https://nodejs.org/en/blog/npm/peer-dependencies/ that "Now, when installing chai-as-promised, the chai package will come along with it," but no matter because that's obviously incorrect), I don't have a problem with listing it as a peerDependency.

So... I guess in the end my only thought would be to suggest that we use our npm version of React in our demos, and get rid of the bower version. I see https://github.com/jxnblk/rebass does this as well. It might be a good chance to explore how a consumer (us, pretty soon) really *should* be using this package.